src/components/LandingImage/LandingImage.jsx

```diff
@@ -77,7 +78,7 @@ function LandingImage() {
             }
           >
             Registration
```

Comment:
Would be nice, if the button stays there. Could you re-add the link to the images?